nebula-spark-connector/src/main/scala/com/vesoft/nebula/connector/writer/NebulaEdgeWriter.scala

```diff
@@ -78,12 +82,14 @@ class NebulaEdgeWriter(nebulaOptions: NebulaOptions,
     val nebulaEdges = NebulaEdges(propNames, edges.toList, srcPolicy, dstPolicy)
     val exec = if (nebulaOptions.writeMode == WriteMode.INSERT) {
       NebulaExecutor.toExecuteSentence(nebulaOptions.label, nebulaEdges)
-    } else {
+    } else if (nebulaOptions.writeMode == WriteMode.UPDATE) {
       nebulaEdges.values
         .map { edge =>
           NebulaExecutor.toUpdateExecuteStatement(nebulaOptions.label, nebulaEdges.propNames, edge)
         }
         .mkString(";")
+    } else {
```

Comment:
prefer switch, or using 
```
else if ( nebulaOptions.writeMode == WriteMode.DELETE){
}
else{ Exception}
```

it maybe more explicit

Reply:
That's right, in https://github.com/vesoft-inc/nebula-spark-utils/pull/134/ I'll use switch.